dd-sdk-android/src/test/kotlin/com/datadog/android/error/internal/CrashReportsFeatureTest.kt

```diff
@@ -253,6 +257,51 @@ internal class CrashReportsFeatureTest {
         assertThat(endpointUrl).isSameAs(endpointUrl2)
     }
 
+    @Test
+    fun `it will register and unregister the provided plugin when initialise and stop called`(
```

Comment:
Just to be safe, I'd split this in 2 tests : verifying that initialize only calls register, and stop only calls unregsiter.